### DISCUSSION TITLE
Utilize Photo timestamp instead of upload time

### DISCUSSION THREAD

**Discussion by mpking828** [marcbelmont/deep-license-plate-recognition#209] [Q&A]
I have an offline camera where I must manually retrieve the memory card of the camera, and post process the images.

Is there an option to use the Date/Timestamp embedded in the file instead of the time it was uploaded?

**Comment by danleyb2** [comment]
@mpking828 unfortunately we don't support this but you can easily modify the upload script(`plate_recognition.py`) to use the timestamp in the file
